src/main/java/org/threadly/concurrent/benchmark/AbstractSchedulerBenchmark.java

```diff
@@ -75,4 +83,23 @@ protected void runTest() throws InterruptedException {
       p.dump(System.out);
     }
   }
+  
+  /**
+   * Small function to simulate threads actually doing something.
+   * 
+   * @param startReferenceTime Time thread started running (should be collected at absolute start)
+   * @param randBool Random boolean that was generated BEFORE execution started
+   */
+  protected void doThreadWork(long startReferenceTime, boolean randBool) {
```

Comment:
The simulated work is in one place now.  Math.pow and Math.log1p just seemed the easiest CPU specific operations we could do (ie things without lots of memory load).